wheatley/bot.py

```diff
@@ -149,7 +168,8 @@ def _check_number_of_bells(self) -> bool:
                                 + f"but the current tower has {self._tower.number_of_bells}. "
                                 + "Wheatley will not ring!")
             return False
-        if self._tower.number_of_bells > self.row_generator.stage + 1:
+        if (self._tower.number_of_bells > self.row_generator.stage + 1
+                and self.row_generator.custom_start_row is None):
```

Comment:
Bit of a nitpick, but it's probably clearer to finish with a `\` instead of using brackets:
```python
        if self._tower.number_of_bells > self.row_generator.stage + 1 \
                and self.row_generator.custom_start_row is None:
```

Reply:
Cool, I picked this up for some python learning so useful tip :)